### DISCUSSION TITLE
Solidity 101: are the transactions EIP-1559 compatible?

### DISCUSSION THREAD

**Discussion by encrypto95** [Cyfrin/foundry-full-course-cu#2711] [Q&A] [answered]
Hello!

I would be very grateful if the community could aid in my understanding. I'm sorry if this has already been answered, I looked for EIP-1559 and couldn't find anything.

There is the following slide regarding transaction fields in the course solidity smart contract development, section 3 fund me, 4. solidity reverts at 3minutes:

![image](https://github.com/user-attachments/assets/3c7dbb02-072c-497e-984b-fff83eab4b31)

I learned on the ethereum.org website about the EIP-1559 (and there after the EIP-2718) [here](https://ethereum.org/en/developers/docs/transactions/), where there exists these `maxPriorityFeePerGas` and `maxFeePerGas` parameters. I understand what these mean, also [previously discussed](https://github.com/Cyfrin/foundry-full-course-cu/discussions/1254).

My question is, are the aforemention parameters in the slides an example of an EIP-1559 transaction? if so, how? if not, what type of transaction is it and how can we relate/convert it to EIP-1559 transactions?

Thank you!

encrypto

**Comment by EngrPips** [comment, marked as answer]
Hello @encrypto95, The slide is just telling you what it entails to make a transaction on the Ethereum blockchain. If you carefully read the parameters in the slide, you will see that it is just telling the things that make up a transaction. What type of transaction you want to make is up to you, and how to make the type of transaction you want to make requires you to read to the eip for that specific transaction.

**Reply by encrypto95** [reply]
Thanks for the answer @EngrPips! I think I got confused by the v, r and s fields and I dedicated them to being a specific transaction.

It's clear!

**Reply by EngrPips** [reply]
Awesome, I am glad it is clear now. The `v`, `r`, and `s` are the signature part of every transaction. Maybe an account abstraction wallet can be excluded in some cases but that is a story for another day lol.
